packages/components/src/types/site.ts

```diff
@@ -13,45 +13,54 @@ import { NotificationSchema } from "~/interfaces/internal/Notification"
 
 export const SiteConfigSchema = Type.Object({
   siteName: Type.String({
-    title: "Name of the site",
-    description: "The name of the site, displayed in the footer.",
+    title: "Site name",
+    description:
+      "This is displayed on browser tabs, the footer, and the Search Results page. It’s also the default meta title of your homepage.",
   }),
   url: Type.String({
     title: "Base URL of the site",
     description: "The base URL of the site.",
+    format: "hidden",
   }),
   agencyName: Type.Optional(
     Type.String({
-      title: "Agency name",
-      description:
-        "The name of the agency, displayed in the copyright footer for non-Government websites.",
+      title: "Website is owned by",
+      description: "This isn't displayed anywhere on your site",
+      format: "disabled",
     }),
   ),
```

Comment:
question(non-blocking): this one is a bit weird in the sense that it should be required if `isGovernment` is set to false, because that is when it is shown at the copyright text in the footer

But I know we don't really have the ability to show/hide fields based on the inputs of another, so as long as UI side is ok with this then I'm ok